modules/core/src/main/java/org/apache/ignite/session/SessionContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.session;
+
+import org.apache.ignite.Ignite;
+import org.apache.ignite.resources.SessionContextProviderResource;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Provides access to attributes set with {@link Ignite#withApplicationAttributes}.
+ *
+ * @see SessionContextProviderResource
+ */
+public interface SessionContext {
+    /**
+     * @param name Attribute name.
+     * @return Attribute value, or {@code null} if not speicifed.
+    .*/
+    public @Nullable String getAttribute(String name);
```

Comment:
Users can't get all set of available attributes. Was it intentional?

Reply:
Yes.